liquibase-dist/src/main/install4j/liquibase.install4j

```diff
@@ -0,0 +1,510 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<install4j version="8.0.3" transformSequenceNumber="8">
+  <directoryPresets config="../../../target/liquibase-3.8.6-local-SNAPSHOT/liquibase" />
```

Comment:
Would be nice if this did not have a hardcoded version number (if possible).

Reply:
Good noticing. I had that fixed before, but the install4j ide swapped it out. I set it back.